.github/workflows/build-packages.yml

```diff
@@ -60,6 +60,18 @@ jobs:
           submodules: 'true'
       - uses: hdl/conda-ci@master
 
+  #73
```

Comment:
you have to adjust the num of jobs variable

Reply:
Hmm, the current number is 69, but the last job # is 73 (including prjoxide), should the variable be adjusted to 73 or just incremented to 70?

Reply:
73

Reply:
Ok, done

Reply:
Apparently, there are three jobs missing, as https://github.com/hdl/conda-eda/actions/runs/964458839 was hanging forever. I guess this was because the master build was waiting for those three inexistent jobs to finish. I put it back to 70